### PR TITLE
Feature/build html css

### PR DESCRIPTION
**Pull request by Jayquintana**
### What (if any) features are you implementing?
- html build and CSS comp build out 
### What (if anything) did you refactor?
- no
### Were there any issues that arose?
- no
### Is there anything that you need from your teammate?
- review code, see if we can refactor
### Refactor checklist
- [] Check variable and parameter names: are they descriptive and unique?
- [] Is the code DRY? SRP?
- [] Is it easy for others to read your code? (explanatory comments needed?)
- [] Is the syntax consistent? (e.g. use of semicolons, blank lines)
- [] Are commit messages formatted correctly?

### Any other comments, questions, or concerns?
start iteration 1!

### CODE REVIEW

**Review by jasmyn2244 — APPROVED** [review]
Love it! If we end up with extra time, I like Jacob's ideas of refactoring and grouping out CSS into sections based on what they are styling. I'd love to get that "start activity" button on the right side at some point.

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  box-sizing: border-box;
+  justify-content: space-around;
+
+
+}
+
+.button-categories {
+  display: flex;
+  flex-direction: row;
+  /* height: 20%; */
```
Comment (ectweitmann) [inline styles.css]: Let's delete this commented out code

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+}
+
+input {
+  color: #CBC9CF;
+  border: none;
+  background-color: #46424D;
+  border-bottom: 2px solid  #CBC9CF;
+  margin-top: 5%;
+  /* font-size: 20px;
+  height: 20%; */
```
Comment (ectweitmann) [inline styles.css]: Let's delete these comments too!

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+  text-align: justify;
+
```
Comment (ectweitmann) [inline styles.css]: Let's delete this extra white space too!

**Review by ectweitmann — CHANGES_REQUESTED** [review]

**Pull request merged**